examples/breakout-rooms/src/components/BreakoutStarter/BreakoutStarter.css

```
@@ -21,7 +21,7 @@
   width: 100%;
 }
 
-.starter-action h5 {
+.starter-action .starter-action-title {
```

Double selector isnt needed right?
```suggestion
.starter-action-title {
```